api/src/accounts.rs

```diff
@@ -223,7 +196,7 @@ impl Account {
             .move_struct_fields(&typ, data)?)
     }
 
-    fn account_state(&self) -> Result<AccountState, Error> {
+    fn account_state_impl(&self) -> Result<AccountState, Error> {
```

Comment:
why add the `_impl`? it seems like it might be the only one with such a title :P

Reply:
Good point - I had `account_state` in previous iteration, which has been removed now.